analyzers/its/config/SonarLintExclusions/SonarLint.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AnalysisInput>
+    <Rules>
+    </Rules>
+    <Settings>
+        <Setting>
+            <Key>sonar.cs.ignoreHeaderComments</Key>
+            <Value>true</Value>
+        </Setting>
+        <Setting>
+            <Key>sonar.vbnet.ignoreHeaderComments</Key>
```

Comment:
I think that both of the properties should refer to a single language
```suggestion
            <Key>sonar.cs.ignoreHeaderComments</Key>
```

Reply:
I see that this is how it is in all the SonarLint.xml config - I'll drop a question to understand if the vbnet case is really needed.

Reply:
We can leave as is - that's how it is in the SonarLint.xml